src/i18n.js

```diff
@@ -45,6 +45,6 @@ export const i18n = createI18n({
     locale: localStorage.locale || "en",
     fallbackLocale: "en",
     silentFallbackWarn: true,
-    silentTranslationWarn: true,
+    silentTranslationWarn: process.env.NODE_ENV !== "development",
```

Comment:
Maybe better to add another env var to determine this. Sometimes there are too many language warnings that push out other important debug messages.

Reply:
Which important debug messages?
![image](https://user-images.githubusercontent.com/905878/135027733-834203db-a73b-458f-be6e-594254d3686d.png)

You can reduce warnings by switching to English